packages/react/src/components/VideoPlayer/__stories__/VideoPlayer.stories.js

```diff
@@ -51,3 +45,41 @@ export const Default = ({ parameters }) => {
     </div>
   );
 };
+
+export const aspectRatio1x1 = ({ parameters }) => {
+  return <Default parameters={parameters} />;
+};
+
+aspectRatio1x1.story = {
+  name: 'Aspect ratio 1:1',
+  parameters: {
+    knobs: {
```

Comment:
Given there is no parameter controlled by a knob here:

```suggestion
    props: {
```